### PR TITLE
Update UG Section On Internship Equality 

### PR DESCRIPTION
**Pull request by HM33-Stu**
Made another PR for this due to accidental merge of previous draft PR (#256)
Building on previous PR since changes have already been merged

Related PR (actual change to code): #257

### CODE REVIEW

**Comment by HM33-Stu** [comment]
Introduced into the tutorial section:
image

**Comment by HM33-Stu** [comment]
This will be marked ready for review once #257 has been approved and merged

**Comment by HM33-Stu** [comment]
@LWS49 Added a question inside FAQ as well with link to that section
image

**Comment by HM33-Stu** [comment]
**Changes Made**
Application Status field added back to What Makes An Internship Unique

File: docs/UserGuide.md [hunk 1]
```diff
@@ -809,6 +810,9 @@ Only do so if you are an experienced user! <br>
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Do the following steps: <br> 1. Navigate to the folder where CareerSync.jar is stored. <br> 2. Copy your data file from data/internshipdata.json. <br> 3. Install the app in the other computer and overwrite the empty data file (data/internshipdata.json) it creates with the file that contains the data (JSON file) of your previous CareerSync home folder.
 
+**Q**: Why is my internship entry considered a duplicate?<br>
+**A**: Your new entry has the same value as an existing entry for one or more fields. For more information, please go [here](#must-know-what-makes-an-internship-unique).
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:814]: Perhaps under Add / Edit Command Common errors can have a hyper link here

"If you are facing duplicate internship errors, please refer to FAQ[hyperlinked] point 2."

**Review by CaptainKeqing — COMMENTED** [review]
nit

File: docs/UserGuide.md [hunk 2]
```diff
@@ -307,23 +307,24 @@ Upon reopening the app:<br>
 You now see all your internships that you have when you exited. Your data was saved!
 </div>
 
-<div markdown="span" class="alert alert-danger">
-⚠️ **BEFORE YOU BEGIN!:** <br>
+#### Must-Know: What Makes An Internship Unique
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:310]: since we are not highlighting this point, can remove Must-Know frmo the name

File: docs/UserGuide.md [hunk 3]
```diff
@@ -307,23 +307,24 @@ Upon reopening the app:<br>
 You now see all your internships that you have when you exited. Your data was saved!
 </div>
 
-<div markdown="span" class="alert alert-danger">
-⚠️ **BEFORE YOU BEGIN!:** <br>
+#### What Makes An Internship Unique
```
Comment (HM33-Stu) [inline docs/UserGuide.md:310]: Removed "Must-Know"

File: docs/UserGuide.md [hunk 4]
```diff
@@ -415,10 +416,14 @@ Lets you add an internship entry to be tracked by CareerSync!
...
 
-1. If you miss out any of the mandatory fields, the command will be rejected with an error message.
-Make sure to refer to the error message displayed for the correct format to use!<br>
+1. If you miss out any of the mandatory fields, the command will be rejected with an error message. Make sure to refer to the error message displayed for the correct format to use!<br>
 2. If you input an invalid status, location or email format, the command will be rejected with an error message highlighting the valid values. <br>
+<<<<<<< Updated upstream
 3. When using the `add` command, the `/phone` parameter allows for more digits that are necessary. Do manually ensure that the length of the phone numbers you input are correct.
+=======
+3. If you encounter the **This internship already exists in the internship list** message, please refer to our [FAQ](#faq) to learn about what makes an internship unique.
+
```
Comment (HM33-Stu) [inline docs/UserGuide.md:425]: Updated the link to the FAQ from add section

File: docs/UserGuide.md [hunk 5]
```diff
@@ -415,10 +416,14 @@ Lets you add an internship entry to be tracked by CareerSync!
...
 
-1. If you miss out any of the mandatory fields, the command will be rejected with an error message.
-Make sure to refer to the error message displayed for the correct format to use!<br>
+1. If you miss out any of the mandatory fields, the command will be rejected with an error message. Make sure to refer to the error message displayed for the correct format to use!<br>
 2. If you input an invalid status, location or email format, the command will be rejected with an error message highlighting the valid values. <br>
+<<<<<<< Updated upstream
 3. When using the `add` command, the `/phone` parameter allows for more digits that are necessary. Do manually ensure that the length of the phone numbers you input are correct.
+=======
+3. If you encounter the `This internship already exists in the internship list` message, please refer to our [FAQ](#faq) to learn about what makes an internship unique.
+
```
Comment (HM33-Stu) [inline docs/UserGuide.md:425]: Added in the link to FAQ from add section

File: docs/UserGuide.md [hunk 6]
```diff
@@ -415,10 +416,10 @@ Lets you add an internship entry to be tracked by CareerSync!
 <div markdown="span" class="alert alert-danger">
 ⚠️ **Common errors:** <br>
 
-1. If you miss out any of the mandatory fields, the command will be rejected with an error message.
-Make sure to refer to the error message displayed for the correct format to use!<br>
+1. If you miss out any of the mandatory fields, the command will be rejected with an error message. Make sure to refer to the error message displayed for the correct format to use! <br>
 2. If you input an invalid status, location or email format, the command will be rejected with an error message highlighting the valid values. <br>
-3. When using the `add` command, the `/phone` parameter allows for more digits that are necessary. Do manually ensure that the length of the phone numbers you input are correct.
+3. When using the `add` command, the `/phone` parameter allows for more digits that are necessary. Do manually ensure that the length of the phone numbers you input are correct. <br>
+4. If you encounter the `This internship already exists in the internship list` message, please go [here](#what-makes-an-internship-unique) to learn about what makes an internship unique.
```
Comment (HM33-Stu) [inline docs/UserGuide.md:422]: Added in the link to FAQ from add section

File: docs/UserGuide.md [hunk 7]
```diff
@@ -513,7 +514,8 @@ Lets you edit an existing internship entry in CareerSync.
 
 <div markdown="span" class="alert alert-danger">
 ⚠️ **Common errors:** <br>
-1. Make sure not to miss out the mandatory `INDEX` field. If you do, CareerSync won't know which internship to edit!
+1. Make sure not to miss out the mandatory `INDEX` field. If you do, CareerSync won't know which internship to edit! <br>
+2. If you encounter the `This internship already exists in the internship list` message, please go [here](#what-makes-an-internship-unique) to learn about what makes an internship unique.
```
Comment (HM33-Stu) [inline docs/UserGuide.md:518]: Added in the link to FAQ from edit section

**Review by CaptainKeqing — APPROVED** [review]
Changes look great to me. Thank you !

**Pull request merged**